performance/Makefile

```diff
@@ -75,18 +75,22 @@ unpersist:
 
 run:
 	docker run \
+		--network=host \
```

Comment:
network=host on docker for mac is busted so i'm not convinced this would work locally for mac devs.

we could instead start this in bridge mode and set `$DET_MASTER=host.docker.interal` for mac and the host ip for linux?

Reply:
I made ```network=host``` only be used in CI and left DET_MASTER up to the user to set if they are running locally